### ISSUE TITLE
Alt+left/right shows file menu on Linux Mint

### ISSUE THREAD

**Issue by ham1** [brave/browser-laptop#1210]
Using:
v0.8.2 on Linux Mint 17.3

When navigating around using Alt+left/right (back/forwards) the file menu is shown when releasing the alt key which is annoying because it hides (and subsequently moves the page) when you try to click on a link.

Steps:
1. Navigate to https://github.com/brave/browser-laptop/issues
2. Click a link
3. Press Alt+left (to go back)
4. Click a link on issues page

Expected: Link is clicked and opens final issue page.
Actual:
After step 3, the file menu bar appears link in this screenshot:
![image](https://cloud.githubusercontent.com/assets/3393038/14182519/6da67a48-f763-11e5-8db4-57662bc09acc.png)
When trying to click a link (step 4), instead of clicking the link the file menu hides and the link is not clicked (because it moves up relative to the mouse position)

**Comment by luixxiul** [comment]
Would you please update to the latest version and try again? Thanks!

**Comment by ham1** [comment]
Just tests with 0.10.0 and this still happens. I'm on Mint 17.3 if that helps.

**Comment by bsclifton** [comment]
Alt will toggle the menu on Windows and versions of Linux not using Unity (since Unity always shows the menu and you can't hide it). Just pushing alt should toggle it

**Comment by ham1** [comment]
The same thing does not happen in Firefox for example.
When pressing (and then releasing) just the `alt` key, yes, the file menu appears in Firefox and Brave which is as expected.

However, when pressing down the `alt` key, holding it, **and** then pressing the left key, then releasing both. Both browsers navigate back but Firefox does **not** display the file menu.

**Comment by bsclifton** [comment]
gotcha- thanks for clarifying :smile:

**Comment by ham1** [comment]
Keep up the good work guys. Just fFYI, still happening on v0.10.3.

**Comment by Liunkae** [comment]
Related issues: #5540 and #5775

**Comment by bsclifton** [comment]
Closing this issue as a duplicate of https://github.com/brave/browser-laptop/issues/5540

Please subscribe to that issue to get updates. Thanks! 😄
